edgelet/hsm-sys/azure-iot-hsm-c/inc/hsm_client_data.h

```diff
@@ -281,6 +332,8 @@ typedef struct HSM_CLIENT_X509_INTERFACE_TAG
     HSM_CLIENT_GET_CERT_KEY hsm_client_get_key;
     HSM_CLIENT_GET_COMMON_NAME hsm_client_get_common_name;
     HSM_CLIENT_FREE_BUFFER hsm_client_free_buffer;
+    HSM_CLIENT_SIGN_WITH_PRIVATE_KEY hsm_client_sign_with_private_key;

```

Comment:
Did you update hsm_validator?

Reply:
Thanks for mentioning this. No I have not and I cannot include it in this PR since the validator is in the HSM repo. I will make a note to have that updated as well when these changes finalize and I merge these additions to the interface to the HSM repo.